src/main/kotlin/org/jetbrains/research/libsl/nodes/statements.kt

```diff
@@ -20,4 +70,32 @@ data class Action(
         append(args.joinToString(separator = ", "))
         append(");")
     }
-}
\ No newline at end of file
+}
+
+data class Proc(
```

Comment:
As far as I know, this class describes a procedure call. So, could you rename it to something like `ProcedureCall` or `ProcedureInvoke` or `ProcedureInvocation`?